polars/polars-core/src/series/implementations/dates_time.rs

```diff
@@ -308,7 +308,7 @@ macro_rules! impl_dyn_series {
                         .into_series()
                         .date()
                         .unwrap()
-                        .strftime("%Y-%m-%d")
+                        .to_string("%Y-%m-%d")
```

Comment:
Which then makes these changes unneeded.

Reply:
I thought it would be nice to use the more readable / more ideally named version internally, and keep `strftime` for Polars users who have familiarity with that name.

But indeed, technically we can keep using `strftime` here if we keep those methods on the chunked arrays.

Reply:
RUST SIDE:

Do we now have `strftime` and `to_string` as expression? I think we should keep both on the expression side to keep consistency with python. 

In the lower level implementation, I think it's fine to only keep `to_string`.

Reply:
> RUST SIDE:
> 
> Do we now have `strftime` and `to_string` as expression? I think we should keep both on the expression side to keep consistency with python.

We do! `DateLikeNameSpace` has both `to_string` and `strftime` implemented.

And for Series, `TemporalMethods` also has both. So that should cover it.